src/sentry/integrations/vercel/client.py

```diff
@@ -39,8 +43,22 @@ def get_user(self):
         return self.get_cached(self.USER_URL)["user"]
 
     def get_projects(self):
-        # TODO: we will need pagination since we are limited to 20
-        return self.get(self.PROJECTS_URL)["projects"]
+        limit = 20
```

Comment:
it says in the docs that the default is `20`, any chance that we tried doing more ?

Reply:
@MeredithAnya No, but I don't want to rely on logic that doesn't match the docs. Rather play it safe :)